_posts/2019-10-24-writing-a-compatible-date-input.md

```diff
@@ -15,13 +15,32 @@ image_url: https://unsplash.com/photos/FoKO4DpXamQ
+Working with dates can seem a little daunting sometimes. We need correct formatting, and hopefully we don't have to deal with timezones at all. Dates come up a lot though, so it's important to be able to work with them. Luckily, we have the handy `<input type="date">`, which we can just use and have things nicely formatted and know that the user is going to provide the date in an expected format.
+
+## The Problem
+
+We decided that we can use `<input type="date">` to easily tackle this issue, right? Unfortunately, if you have to support legacy browsers at all, it isn't that simple. Internet Explorer and Safari do not support date inputs.
+
+## The Solution
+
+I'll break down the solution and why it works, but if you'd like to get a look at the code and play with it yourself, there is a CodePen demo available.
+
+[**WORKING DEMO ON CODEPEN**](https://codepen.io/jaime-lynn/pen/MWWmrbN)
 
 Luckily, the solution to this is relatively painless. Here at RIMdev, we use Vue.js, and so our solution will examine how to write a simple date input with our favorite front end framework, but this kind of behavior could absolutely be translated to other use cases.
 
-The important part to note here is that even if you have `<input type="date">` in Internet Explorer or Safari, it will render as `<input type="text">`. This allows you to check whether or not the user's browser supports date inputs without having to do any kind of checking about what browser the user is on. If the input is a date input, you can simply exit out of any additional methods (or run anything you would, such as additional events) and move on. If the input is a text input, then we can fall back into some good old fashioned regex to trim and format the date input as the user types. A caveat to this - if you are checking to format the input on any type of keyup or keypress event, it is a good idea to ignore the backspace and delete actions so that the user doesn't run into unintended formatting as they are clearing the input.
+**The important part to note here is that even if you have `<input type="date">` in Internet Explorer or Safari, it will render as `<input type="text">`.**
+
+This allows you to check whether or not the user's browser supports date inputs without having to do any kind of checking about what browser the user is on. If the input is a date input, you can simply exit out of any additional methods (or run anything you would, such as additional events) and move on. If the input is a text input, then we can fall back into some good old fashioned regex to trim and format the date input as the user types.
```

Comment:
When you use the word `This` it can come off as vague. Try using the subject or ask yourself what you mean by `This`